featuregate/client/pkg/featuregateclient/client.go

```diff
@@ -1,21 +1,20 @@
 // Copyright 2021 VMware, Inc. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0
 
-package client
+package featuregateclient
```

Comment:
`featuregate/client/pkg/featuregateclient` feels too repetitive? Any suggestions for a better package name?

Reply:
I liked your `gate` alias. Maybe it can be `gateclient`. And instead of the method `NewFeatureGateClient`, it can just be `New` so that it doesn't stutter.